central/cve/datastore/singleton.go

```diff
@@ -28,24 +15,10 @@ var (
 
 func initialize() {
 	globalDackbox := globaldb.GetGlobalDackBox()
```

Comment:
Why do we still need dackbox here?  Is this one of those CVE things we do differently now that needs to be deprecated itself?

Reply:
This PR does not remove stores, only indexers and searchers :)

Reply:
one step at a time.  got it.